drivers/ZW_Dimmer_Rotary/device.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const { ZwaveDevice } = require('homey-meshdriver');
+
+const util = require('./../../node_modules/homey-meshdriver/lib/util');
+
+module.exports = class ZW_DimmerRotary extends ZwaveDevice {
+	onMeshInit() {
+
+		// enable debugging
```

Comment:
I suggest commenting out lines 11 and 14.

Reply:
If I'm correct, this is in here for a purpose. It's difficult to debug remotely from crash reports having no info. Having at least some data would provide useful log statements. Homey users will never see this anyways?

Reply:
You're probably right 😄